src/seedu/addressbook/commands/FilterCommand.java

```diff
@@ -0,0 +1,34 @@
+package seedu.addressbook.commands;
+
+import seedu.addressbook.data.person.ReadOnlyPerson;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class FilterCommand extends Command {
+  public static final String COMMAND_WORD = "filter";
+  public static final String MESSAGE_USAGE = COMMAND_WORD + ": Finds all person whose names begin with the specified alphabet.\n"
```

Comment:
What yunjun said - watch out for line character limit for adherence to coding standard